### DISCUSSION TITLE
Got issue when using OpenSearch

### DISCUSSION THREAD

**Discussion by genwch** [FlowiseAI/Flowise#585] [General]
when using OpenSearch in LangChain, some parameters can be set as below
e.g. http_auth, verify_certs

``` python
    docsearch = OpenSearchVectorSearch.from_documents(
        docs,
        embeddings,
        opensearch_url=opensearch_url,
        http_auth=("admin", "admin"),
        verify_certs=False,
        index_name="flowise-test-index-1",
    )
```

how can I set it when using Flowise?
<img width="853" alt="Screenshot 2023-07-20 at 1 59 20 PM" src="https://github.com/FlowiseAI/Flowise/assets/18664789/ce1898e6-fe73-4022-af7b-ab0310343d7f">

**Comment by chungyau97** [comment]
Hi @genwch, 

I believe you're referencing LangChain Python.
I looked through the source code of LangChain JS; It's not yet implemented yet.

References:
[LangChain JS](https://github.com/hwchase17/langchainjs/blob/main/langchain/src/vectorstores/opensearch.ts)
[LangChain JS Docs](https://js.langchain.com/docs/modules/data_connection/vectorstores/integrations/opensearch)
[LangChain Python Docs](https://python.langchain.com/docs/modules/data_connection/vectorstores/integrations/opensearch)
